### DISCUSSION TITLE
Change node_modules location

### DISCUSSION THREAD

**Discussion by kursatsaka** [storybookjs/storybook#13897] [Help]
I have a Angular 10 project. I want to add storybook on my project but my node_modules folder in parent folder. How can i change node_modules location for storybook.

Thanks for your help.

**Comment by gaetanmaisse** [comment]
I'm not sure to understand what you're trying to achieve, plus, the dependencies in `node_modules` are managed by npm/yarn and not Storybook itself so we won't be able to help on that side. Can you elaborate and provide a repro repo?
